db/migrate/20180925100351_create_users.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :users do |t|
+      t.string :email
+      t.string :password
```

Comment:
Do we want a name and handle for Users or are we okay with just email and password?